python/tchannel/tornado/data.py

```diff
@@ -172,6 +177,13 @@ def status_code(self, status):
 
         self.flags = status.value
 
+    @property
+    def ok(self):
+        if self.flags == StatusCode.ok.value:
```

Comment:
`return self.flags == StatusCode.ok.value`

When I think of flags I usually think of a bitmask, so it's strange that you're using equality here. Don't we also have a `code` attribute?